fe/fe-core/src/main/java/org/apache/doris/nereids/util/FrontendConjunctsUtils.java

```diff
@@ -0,0 +1,123 @@
+// under the License.
+
+package org.apache.doris.nereids.util;
+
+import org.apache.doris.analysis.Expr;
+import org.apache.doris.nereids.analyzer.UnboundSlot;
+import org.apache.doris.nereids.parser.NereidsParser;
+import org.apache.doris.nereids.rules.expression.rules.FoldConstantRuleOnFE;
+import org.apache.doris.nereids.trees.expressions.Expression;
+import org.apache.doris.nereids.trees.expressions.literal.BooleanLiteral;
+import org.apache.doris.nereids.trees.expressions.literal.Literal;
+import org.apache.doris.persist.gson.GsonUtils;
+
+import com.google.common.collect.Maps;
+import com.google.gson.reflect.TypeToken;
+import org.apache.commons.collections.CollectionUtils;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.concurrent.atomic.AtomicBoolean;
+import java.util.stream.Collectors;
+
+/**
+ * FrontendConjunctsUtils
+ */
+public class FrontendConjunctsUtils {
+    private static final Logger LOG = LogManager.getLogger(FrontendConjunctsUtils.class);
+
+    public static List<Expression> convertToExpression(String conjuncts) {
+        List<Expr> exprs = GsonUtils.GSON.fromJson(conjuncts, new TypeToken<List<Expr>>() {
+        }.getType());
+        return exprs.stream()
+                .map(expr -> exprToExpression(expr))
+                .collect(Collectors.toList());
+    }
+
+    public static Expression exprToExpression(Expr expr) {
+        NereidsParser nereidsParser = new NereidsParser();
+        return nereidsParser.parseExpression(expr.toSql());
+    }
+
+    public static boolean isFiltered(List<Expression> expressions, String columnName, Object value) {
+        HashMap<String, Object> values = Maps.newHashMapWithExpectedSize(1);
+        values.put(columnName.toLowerCase(), value);
+        return isFiltered(expressions, values);
+    }
+
+    /**
+     * isFiltered
+     *
+     * @param expressions expressions
+     * @param values values
+     * @return isFiltered
+     */
+    public static boolean isFiltered(List<Expression> expressions, Map<String, Object> values) {
+        for (Expression expression : expressions) {
+            if (isFiltered(expression, values)) {
+                return true;
+            }
+        }
+        return false;
+    }
+
+    /**
+     * isFiltered
+     *
+     * @param expression expression
+     * @param values values
+     * @return isFiltered
+     */
+    public static boolean isFiltered(Expression expression, Map<String, Object> values) {
+        try {
+            AtomicBoolean containsAllColumn = new AtomicBoolean(true);
+            Expression rewrittenExpr = expression.rewriteUp(expr -> {
+                if (expr instanceof UnboundSlot) {
+                    List<String> nameParts = ((UnboundSlot) expr).getNameParts();
+                    if (!CollectionUtils.isEmpty(nameParts) && nameParts.size() == 1) {
```

Comment:
why need `nameParts.size() == 1`? maybe just get the last part from name is enough